plugins/builds/steps/update.js

```diff
@@ -57,9 +57,6 @@ module.exports = () => ({
                 .then(build => reply(build.steps[stepIndex]).code(200))
                 .catch(err => reply(boom.wrap(err)));
         },
-        response: {
```

Comment:
I might be missing context on this, but why was this particular schema dropped?

Reply:
I may be wrong but it seems like this response schema was intended only for GETs on this endpoint.

Reply:
I think the schema still applies to the PUT endpoint; they were probably just reusing it

Reply:
ooh, my mistake, I will remove that commit!